src/Runner.Worker/FileCommandManager.cs

```diff
@@ -364,6 +352,18 @@ public EnvFileKeyValuePairs(IExecutionContext context, string filePath)
 
                         output = endIndex > startIndex ? text.Substring(startIndex, endIndex - startIndex) : string.Empty;
                     }
+                    // Normal style NAME=VALUE
+                    else if (equalsIndex >= 0 && heredocIndex < 0)
```

Comment:
Now we can be stricter here:

Note that we're checking

```
if (equalsIndex >= 0 && heredocIndex < 0)
```

whereas before it was:
```
if (equalsIndex >= 0 && (heredocIndex < 0 || equalsIndex < heredocIndex))
```